diff -up plasma-workspace-master/wallpapers/image/main.xml.in.omv~ plasma-workspace-master/wallpapers/image/main.xml.in
--- plasma-workspace-master/wallpapers/image/main.xml.in.omv~	2023-09-27 22:02:15.711890601 +0200
+++ plasma-workspace-master/wallpapers/image/main.xml.in	2023-09-27 22:02:27.474362175 +0200
@@ -20,7 +20,7 @@
     </entry>
     <entry name="Image" type="String">
       <label>Wallpaper image path or wallpaper name</label>
-      <default>@KDE_INSTALL_FULL_WALLPAPERDIR@/Next/</default>
+      <default>@KDE_INSTALL_FULL_WALLPAPERDIR@/OpenMandriva/</default>
     </entry>
     <entry name="PreviewImage" type="String">
       <label>Wallpaper preview image path</label>
