diff -up plasma-workspace-5.92.0/dataengines/weather/ions/CMakeLists.txt.1~ plasma-workspace-5.92.0/dataengines/weather/ions/CMakeLists.txt
--- plasma-workspace-5.92.0/dataengines/weather/ions/CMakeLists.txt.1~	2024-01-11 00:15:04.438176560 +0100
+++ plasma-workspace-5.92.0/dataengines/weather/ions/CMakeLists.txt	2024-01-11 00:15:33.062428213 +0100
@@ -14,8 +14,8 @@ generate_export_header(weather_ion BASE_
 target_link_libraries (weather_ion PRIVATE KF6::I18n PUBLIC Qt::Core Plasma::Plasma Plasma::Plasma5Support)
 
 set_target_properties(weather_ion PROPERTIES
-   VERSION 7.0.0
-   SOVERSION 7
+   VERSION 8.0.0
+   SOVERSION 8
 )
 
 install (TARGETS weather_ion EXPORT kdeworkspaceLibraryTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
diff -up plasma-workspace-5.92.0/libnotificationmanager/CMakeLists.txt.1~ plasma-workspace-5.92.0/libnotificationmanager/CMakeLists.txt
--- plasma-workspace-5.92.0/libnotificationmanager/CMakeLists.txt.1~	2024-01-10 14:11:11.000000000 +0100
+++ plasma-workspace-5.92.0/libnotificationmanager/CMakeLists.txt	2024-01-11 00:15:04.438176560 +0100
@@ -92,7 +92,7 @@ target_link_libraries(notificationmanage
 
 set_target_properties(notificationmanager PROPERTIES
                       VERSION ${PROJECT_VERSION}
-                      SOVERSION 1
+                      SOVERSION 2
                       EXPORT_NAME LibNotificationManager)
 
 ecm_add_qml_module(notificationmanager URI org.kde.notificationmanager GENERATE_PLUGIN_SOURCE)
diff -up plasma-workspace-5.92.0/libtaskmanager/CMakeLists.txt.1~ plasma-workspace-5.92.0/libtaskmanager/CMakeLists.txt
--- plasma-workspace-5.92.0/libtaskmanager/CMakeLists.txt.1~	2024-01-10 14:11:11.000000000 +0100
+++ plasma-workspace-5.92.0/libtaskmanager/CMakeLists.txt	2024-01-11 00:15:04.438176560 +0100
@@ -74,7 +74,7 @@ endif()
 
 set_target_properties(taskmanager PROPERTIES
                       VERSION ${PROJECT_VERSION}
-                      SOVERSION 6
+                      SOVERSION 7
                       EXPORT_NAME LibTaskManager)
 
 
