diff -up plasma-workspace-master/CMakeLists.txt.1~ plasma-workspace-master/CMakeLists.txt
--- plasma-workspace-master/CMakeLists.txt.1~	2023-07-07 07:57:04.000000000 +0200
+++ plasma-workspace-master/CMakeLists.txt	2023-07-07 14:22:49.505953962 +0200
@@ -6,6 +6,7 @@ string(REPLACE "." ";" VERSION_LIST ${PR
 list(GET VERSION_LIST 0 PROJECT_VERSION_MAJOR)
 list(GET VERSION_LIST 1 PROJECT_VERSION_MINOR)
 list(GET VERSION_LIST 2 PROJECT_VERSION_PATCH)
+set(PROJECT_VERSION_MAJOR 6)
 
 set(QT_MIN_VERSION "6.5.0")
 set(KF6_MIN_VERSION "5.240.0")
diff -up plasma-workspace-master/dataengines/weather/ions/CMakeLists.txt.1~ plasma-workspace-master/dataengines/weather/ions/CMakeLists.txt
--- plasma-workspace-master/dataengines/weather/ions/CMakeLists.txt.1~	2023-07-07 07:57:04.000000000 +0200
+++ plasma-workspace-master/dataengines/weather/ions/CMakeLists.txt	2023-07-07 14:12:09.668901011 +0200
@@ -14,8 +14,8 @@ generate_export_header(weather_ion BASE_
 target_link_libraries (weather_ion PRIVATE KF6::I18n PUBLIC Qt::Core KF6::Plasma KF6::Plasma5Support)
 
 set_target_properties(weather_ion PROPERTIES
-   VERSION 7.0.0
-   SOVERSION 7
+   VERSION 8.0.0
+   SOVERSION 8
 )
 
 install (TARGETS weather_ion EXPORT kdeworkspaceLibraryTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
diff -up plasma-workspace-master/libnotificationmanager/CMakeLists.txt.1~ plasma-workspace-master/libnotificationmanager/CMakeLists.txt
--- plasma-workspace-master/libnotificationmanager/CMakeLists.txt.1~	2023-07-07 07:57:04.000000000 +0200
+++ plasma-workspace-master/libnotificationmanager/CMakeLists.txt	2023-07-07 14:12:09.668901011 +0200
@@ -92,7 +92,7 @@ target_link_libraries(notificationmanage
 
 set_target_properties(notificationmanager PROPERTIES
                       VERSION ${PROJECT_VERSION}
-                      SOVERSION 1
+                      SOVERSION 2
                       EXPORT_NAME LibNotificationManager)
 
 install(TARGETS notificationmanager EXPORT notificationmanagerLibraryTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS} )
diff -up plasma-workspace-master/libtaskmanager/CMakeLists.txt.1~ plasma-workspace-master/libtaskmanager/CMakeLists.txt
--- plasma-workspace-master/libtaskmanager/CMakeLists.txt.1~	2023-07-07 07:57:04.000000000 +0200
+++ plasma-workspace-master/libtaskmanager/CMakeLists.txt	2023-07-07 14:12:09.668901011 +0200
@@ -64,7 +64,7 @@ endif()
 
 set_target_properties(taskmanager PROPERTIES
                       VERSION ${PROJECT_VERSION}
-                      SOVERSION 6
+                      SOVERSION 7
                       EXPORT_NAME LibTaskManager)
 
 
