diff -up plasma-workspace-master/lookandfeel/sddm-theme/Main.qml.omv~ plasma-workspace-master/lookandfeel/sddm-theme/Main.qml
--- plasma-workspace-master/lookandfeel/sddm-theme/Main.qml.omv~	2023-08-15 19:50:25.088812063 +0200
+++ plasma-workspace-master/lookandfeel/sddm-theme/Main.qml	2023-08-15 19:50:50.796966367 +0200
@@ -9,6 +9,7 @@ import QtQuick.Layouts 1.15
 import QtQuick.Controls 2.15 as QQC2
 import Qt5Compat.GraphicalEffects
 
+import org.kde.plasma.core 2.0 as PlasmaCore
 import org.kde.plasma.components 3.0 as PlasmaComponents3
 import org.kde.plasma.plasma5support 2.0 as P5Support
 import org.kde.kirigami 2.20 as Kirigami
